### PR TITLE
add and test GET /api/articles endpoint, tidy up require paths

### PR DESCRIPTION
**Pull request by MichalLewczuk**

### CODE REVIEW

File: __tests__/app.tests.js [hunk 1]
```diff
@@ -34,11 +34,32 @@ describe("TOPICS TESTS", () => {
...
+      expect(res.body.articles).toBeSortedBy("created_at", {
+        descending: true,
+      });
+      res.body.articles.forEach((article) => {
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
```
Comment (jadelandeg) [inline __tests__/app.tests.js]: shouldn't get body when we get all articles

File: models/articles.models.js [hunk 2]
```diff
@@ -1,5 +1,18 @@
 const db = require("../db/connection");
 
+//SELECT
+
+exports.selectArticles = async () => {
+  const result = await db.query(`SELECT articles.*,
```
Comment (jadelandeg) [inline models/articles.models.js]: how can we change this so we don't get the body

**Review by jadelandeg — CHANGES_REQUESTED** [review]
Well done, looking great. Just one thing, check the trello and see what we actually want to get back from articles.....

File: __tests__/app.tests.js (hunk 1 above)
Comment (MichalLewczuk) [reply __tests__/app.tests.js]: I need to read the task description more carefully. Removed now.

File: models/articles.models.js (hunk 2 above)
Comment (MichalLewczuk) [reply models/articles.models.js]: I need to read the task description more carefully. Removed now.

**Pull request merged**